partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnIsMissing.kt

```diff
@@ -5,15 +5,16 @@ package org.partiql.spi.function.builtins
 
 import org.partiql.spi.function.Function
 import org.partiql.spi.function.Parameter
+import org.partiql.spi.function.utils.FunctionUtils
 import org.partiql.spi.value.Datum
 import org.partiql.types.PType
 
 /**
- * Function (operator) for the `IS MISSING` special form.
+ * Function (operator) for the `IS MISSING` special form. Its name is obfuscated via [FunctionUtils.SYSTEM_PREFIX_INTERNAL].
```

Comment:
Similarly for some other comments.

```suggestion
 * Function (operator) for the `IS MISSING` special form. Its name is hidden via [FunctionUtils.SYSTEM_PREFIX_INTERNAL].
```